query/src/sql/optimizer/rule/transform_state.rs

```diff
@@ -14,20 +14,21 @@
 
 use crate::sql::optimizer::s_expr::SExpr;
 
+#[derive(Clone, Default)]
 pub struct TransformState {
     results: Vec<SExpr>,
 }
 
 impl TransformState {
-    pub fn create() -> Self {
+    pub fn new() -> Self {
```

Comment:
If there is no inputs for `new`, how about using `default()` instead?

Reply:
I'm not sure that it won't have any input. Thus I prefer to keep this.